kairon/shared/utils.py

```
@@ -2229,6 +2229,21 @@ def validate_data(form_data):
         if not is_first_valid or not is_last_valid:
             raise AppException("First name and last name can only contain letters, numbers, spaces and underscores.")
 
+    @staticmethod
+    def validate_data(form_data):
```

Ruff (0.8.2)

Redefinition of unused `validate_data` from line 2217

(F811)